app/models/metric/purging.rb

```diff
@@ -61,6 +61,8 @@ def self.purge_count(older_than, interval)
     purge_scope(older_than, interval).count
   end
 
+  # TODO: delete these using the same created_at queries by date
+  # rather than by joining to metrics - then hourly could be truncated vs deleted
```

Comment:
Having more comments around the `truncation` strategy and workflow would allow this comment to reference those and be more meaningful.